### PR TITLE
[DOC] add Anthony as maintainer

### PR DESCRIPTION
**Pull request by Remi-Gau**
@bendhouseart 
same as for the bids-website. :wink: (https://github.com/bids-standard/bids-website/pull/198)

### CODE REVIEW

File: DECISION-MAKING.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ BIDS governance.
 | Franklin Feingold ([@franklin-feingold](https://github.com/franklin-feingold)) | 5h/week         | Community development      |
 | Taylor Salo ([@tsalo](https://github.com/tsalo))                               | 3h/week         | MRI                        |
 | Remi Gau ([@Remi-Gau](https://github.com/Remi-Gau))                            | 3h/week         | Community development, MRI |
+| Anthony Galassi  ([@bendhouseart](https://github.com/bendhouseart))            | 3h/week         | PET, ???                   |
```
Comment (Remi-Gau) [inline DECISION-MAKING.md:34]: @bendhouseart I will let you edit the details of that line. :smile:

Comment (bendhouseart) [reply DECISION-MAKING.md:34]: @Remi-Gau PET is fine for now, can always update. If you think outreach counts as Community development then chuck that in there too.

Comment (Remi-Gau) [inline DECISION-MAKING.md:34]:
```suggestion
| Anthony Galassi  ([@bendhouseart](https://github.com/bendhouseart))            | 3h/week         | PET, Community development                   |
```

File: DECISION-MAKING.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ BIDS governance.
 | Franklin Feingold ([@franklin-feingold](https://github.com/franklin-feingold)) | 5h/week         | Community development      |
 | Taylor Salo ([@tsalo](https://github.com/tsalo))                               | 3h/week         | MRI                        |
 | Remi Gau ([@Remi-Gau](https://github.com/Remi-Gau))                            | 3h/week         | Community development, MRI |
+| Anthony Galassi  ([@bendhouseart](https://github.com/bendhouseart))            | 3h/week         | PET, Community development                   |
```
Comment (Remi-Gau) [inline DECISION-MAKING.md:34]:
```suggestion
| Anthony Galassi  ([@bendhouseart](https://github.com/bendhouseart))            | 3h/week         | PET, Community development |
```

File: DECISION-MAKING.md (hunk 1 above)
Comment (Remi-Gau) [reply DECISION-MAKING.md]: Done. Feel free to press the big green button :arrow_down: .

**Review by sappelhoff — APPROVED** [review]

**Pull request merged**